packages/icons/src/info/Exclamation.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import { SVG } from '@marigold/system';
+import { SVG, SVGProps } from '@marigold/system';
 
-export const Exclamation = ({ className = '', ...props }) => (
-  <SVG className={className} {...props}>
+export const Exclamation = (props: SVGProps) => (
```

Comment:
Should we do this in all Icons? Not in this PR but good point.

Reply:
Guess so :)